docs/src/model_creation/reactionsystem_content_accessing.md

```diff
@@ -0,0 +1,249 @@
+# [Accessing model properties](@id model_accessing)
+Catalyst is based around the creation, analysis, and simulation of chemical reaction network models. Catalyst stores these models in [`ReactionSystem](@ref) structures. This page describes some basic functions for accessing the content of these structures. This includes retrieving lists of species, parameters, or reactions that a model consists of. An extensive lists of relevant functions for working with `ReactionSystem` models can be found in Catalyst's [API](@ref api).
+
+!!! warn
+    Generally, the field of Julia structures can be accessed through `struct.field`. E.g. a simulation's time vector can be retrieved using `simulation.t`. While Catalyst `ReactionSystem`s are structures, one should *never* access their fields using this approach, but rather using the accessor functions described below and in the [API](@ref api_accessor_functions) (direct accessing of fields can yield unexpected behaviours). E.g. to retrieve the species of a `ReactionsSystem` `rs`, use `Catalyst.get_species(rs)`, *not* `rs.species`.
```

Comment:
```suggestion
    Generally, a field of a Julia structure can be accessed through `struct.fieldname`. E.g. a simulation's time vector can be retrieved using `simulation.t`. While Catalyst `ReactionSystem`s are structures, one should *never* access their fields using this approach, but rather using the accessor functions described below and in the [API](@ref api_accessor_functions) (direct accessing of fields can yield unexpected behaviours). E.g. to retrieve the species of a `ReactionsSystem` `rs`, use `Catalyst.get_species(rs)`, *not* `rs.species`.
```

Reply:
Maybe explain why this shouldn't be done by pointing out that instead `rs.t` is used to accessible symbolic variables encapsulated by `rs`?

Reply:
yes, I will add a bit about that

Reply:
The reason is that, as shown [below](@ref model_accessing_symbolic_variables), Catalyst reserves this type of accessing for accessing symbolic variables stored in the system (i.e. `rs.X` refers to the `X` symbolic variable, not a field in `rs` named "X").

Reply:
Sounds good, but maybe say "Catalyst, and more generally ModelingToolkit for any system type, reserves"

Reply:
It isn't really a policy that we made.